### PR TITLE
CHECKED SHOOTER FIX

### PR DESCRIPTION
**Pull request by asc-2127**
Hi everyone

### CODE REVIEW

**Review by itsmichaelyu — CHANGES_REQUESTED** [review]
delete all of this

File: src/main/java/frc/robot/subsystems/indexer/Indexer.java [hunk 1]
```diff
@@ -99,7 +99,9 @@ public void periodic() {
                     intakeBall();
                     state = State.INTAKE1;
                 }
-                break;
+                else {
```
Comment (itsmichaelyu) [inline src/main/java/frc/robot/subsystems/indexer/Indexer.java:102]: why is this here

**Review by itsmichaelyu — CHANGES_REQUESTED** [review]
fix pls

**Comment by itsmichaelyu** [comment]
You fixed the gitignore, but you need to remove the .run files manually.

**Review by itsmichaelyu — APPROVED** [review]

**Review by itsmichaelyu — CHANGES_REQUESTED** [review]
This needs to be removed, but then prolly good.

**Comment by asc-2127** [comment]
gh wont show me what you want deleted

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java [hunk 2]
```diff
@@ -39,7 +38,7 @@ public void periodic() {
         Logger.getInstance().recordOutput("Shooter/ForcedRMP", forcedSetpoint);
         Logger.getInstance().recordOutput("Shooter/ActualRMP", getVelocity());
         Logger.getInstance().recordOutput("Shooter/AtSetpoint", isAtSetpoint());
-
+        io.configurePID(Constants.SHOOTER_P,0, 0);
```
Comment (itsmichaelyu) [inline src/main/java/frc/robot/subsystems/shooter/Shooter.java:41]: oops

**Review by itsmichaelyu — CHANGES_REQUESTED** [review]
oops

**Review by itsmichaelyu — APPROVED** [review]
Looks good!
